training/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# pull base image
+FROM condaforge/miniforge-pypy3
```

Comment:
Use condaforge/miniforge3 base image, since that is the version that we use to install mamba locally also.